.github/workflows/build.yml

```diff
@@ -2,15 +2,18 @@ name: Build
 
 on:
   [push]
+env:
+  DERIVED_DATA_PATH: 'DerivedData'
+  DEVICE: 'iPhone 12 Pro'
 
 jobs:
   build:
     name: Build and test
     runs-on: macos-12
-    env:
-      DERIVED_DATA_PATH: 'DerivedData'
-      DEVICE: 'iPhone 12 Pro'
```

Comment:
Out of curiosity, was it necessary to move `env` to the top? Just want to understand this diff.